src/tigerbeetle/benchmark_load.zig

```diff
@@ -423,7 +423,7 @@ const Benchmark = struct {
         const ms_time = @divTrunc(batch_end_ns - b.batch_start_ns, std.time.ns_per_ms);
 
         if (b.print_batch_timings) {
-            log.info("batch {}: {} tx in {} ms\n", .{
+            log.info("batch {}: {} tx in {} ms", .{
```

Comment:
Finally! :rofl:

Reply:
To bad we aren't actually runnig  _this_ code, and instead use the one from downloaded release!